SPECS/clang/clang-16.spec

```diff
@@ -0,0 +1,273 @@
+%global maj_ver 16
+%global min_ver 0
+%global patch_ver 0
+
+%global clang_binaries \
```

Comment:
will a dev be able to install both clangs side by side?

Reply:
No, it's not setup for this right now. If you install one version while the other is already installed, it'll result in a package upgrade or downgrade. There would be numerous file conflicts with both installed at the same time.